docs/index.rst

```diff
@@ -124,87 +124,87 @@ There are two ways to save the WCS to a file:
 - `Save a WCS object as an ASDF extension in a FITS file`_
```

Comment:
This line should be removed because the target was removed and it's causing the docs build to fail.